test.js

```diff
@@ -46,6 +46,21 @@ describe('Cassandra Migration', function() {
                 assert.equal(executeSpy.args[0][0], expectedQuery.join(' '));
                 executeSpy.reset();
             });
+
+            it('A simple table with compression', function() {
+                db.createTable('users', {
+                    'name': 'varchar'
+                }, {
+                    'primary_key': 'name',
+                    'compression': "{'sstable_compression': 'DeflateCompressor', 'chunk_length_kb': 1024}"
```

Comment:
Personally, I don't mind either way. I believe (but have not tested) that you need to use single quotes instead of double quotes, so strictly speaking, this is not valid JSON to begin with; in which case a string literal might be better. https://docs.datastax.com/en/cassandra/3.0/cassandra/operations/opsConfigCompress.html?hl=compression